src/jobs/pricewatch.js

```diff
@@ -1,25 +1,47 @@
 const priceWatchSchema = require('../schemas/price-watch')
 const jobSchema = require('../schemas/jobs')
 const { fetchPriceEmbed } = require('../utils/prices')
+const { timeNow, getISODate } = require('../utils/helper');
+const { getExcludedGuilds } = require('../utils/jobs')
+const { DEFAULT_PRICE_WATCH_INTERVAL } = require('../constants/constants')
 
 module.exports = ({
   meta: {
     name: 'price-watch',
     description: 'Sends an embed message of the current statistics of SafeMoon',
-    interval: 300 * 1000,
+    interval: 60 * 1000,
```

Comment:
This `interval` should never be less than the `cacheTime` variable, otherwise it'll just output the same data each time.

Reply:
Same as above in the commnet